build_tools/benchmarks/run_benchmarks_on_android.py

```diff
@@ -390,11 +390,48 @@ def main(args):
     benchmark_groups = json.loads(args.execution_benchmark_config.read_text())
     benchmark_group = benchmark_groups.get(args.target_device_name)
     if benchmark_group is None:
-        raise ValueError("Target device not found in the benchmark config.")
-    run_configs = serialization.unpack_and_deserialize(
-        data=benchmark_group["run_configs"],
-        root_type=List[iree_definitions.E2EModelRunConfig],
-    )
+        raise ValueError(
+            "Target device '{}' not found in the benchmark config.".format(
+                args.target_device_name
+            )
+        )
+    if args.shard_index is None:
```

Comment:
I feel like we can factor these out into a common place, maybe `build_tools/benchmarks/common/benchmark_suite.py`?

Reply:
Yeah, I thought the same, but wanted to check if it works first.

Reply:
I refactored the code into a function and put it on benchmark_suite.py as you suggested. PTAL!

Reply:
Left a small comment, but LGTM! Thanks

Reply:
Thanks. I made the changes and also cleaned up the imports.

Note that I imported the `benchmark_suite` module under the identifier `benchmark_suite_module` because `benchmark_suite` is a widely used variable name in the two files.